EventFiltering/PWGHF/HFFilter.cxx

```diff
@@ -184,7 +209,11 @@ struct HfFilter { // Main struct for HF triggers
     helper.setPtRangeSoftPiSigmaC(ptCuts->get(0u, 4u), ptCuts->get(1u, 4u));
     helper.setPtDeltaMassRangeSigmaC(cutsPtDeltaMassCharmReso->get(0u, 6u), cutsPtDeltaMassCharmReso->get(1u, 6u), cutsPtDeltaMassCharmReso->get(0u, 7u), cutsPtDeltaMassCharmReso->get(1u, 7u), cutsPtDeltaMassCharmReso->get(0u, 8u), cutsPtDeltaMassCharmReso->get(1u, 8u), cutsPtDeltaMassCharmReso->get(0u, 9u), cutsPtDeltaMassCharmReso->get(1u, 9u), cutsPtDeltaMassCharmReso->get(2u, 6u), cutsPtDeltaMassCharmReso->get(2u, 7u), cutsPtDeltaMassCharmReso->get(2u, 8u), cutsPtDeltaMassCharmReso->get(2u, 9u));
     helper.setPtRangeSoftKaonXicResoToSigmaC(ptCuts->get(0u, 5u), ptCuts->get(1u, 5u));
-
+    if (activateSecVtx) {
+      helper.setVtxConfiguration(df2, propagateToPCA, useAbsDCA, useWeightedFinalPCA, maxR, maxDZIni, minParamChange, minRelChi2Change);
+      helper.setVtxConfiguration(dfB, propagateToPCA, useAbsDCA, useWeightedFinalPCA, maxR, maxDZIni, minParamChange, minRelChi2Change);
```

Comment:
In the B vertex we should use `useAbsDCA==True` always to avoid errors due to the vertex between a pseudotrack (the D) and a track. In general, I would consider to put all the configurations hard-coded (it is very unlikely that we will change them).
```suggestion
      helper.setVtxConfiguration(dfB, propagateToPCA, true, useWeightedFinalPCA, maxR, maxDZIni, minParamChange, minRelChi2Change);
```